Makefile

```diff
@@ -6,7 +6,7 @@ GIR_FILES = gir-files/Gtk-3.0.gir
 gir : src/auto/mod.rs
 
 src/auto/mod.rs : Gir.toml $(GIR) $(GIR_FILES)
-	$(GIR) -c Gir.toml
+	$(GIR) -c Gir.toml -s
```

Comment:
It doesn't appear in any automation. Locally a sudden slowdown should be easy enough to notice.